packages/test/test-end-to-end-tests/src/test/benchmark/PasTable.all.spec.ts

```diff
@@ -47,10 +40,7 @@ describeNoCompat("PAS Test", () => {
 				this.matrix.insertCols(0, columnSize);
 				for (let i = 0; i < rowSize; i++) {
 					for (let j = 0; j < columnSize; j++) {
-						const id = j.toString() + i.toString();
-						const sharedString: SharedString = createString(id, dataStoreRuntime);
-						sharedString.insertText(0, "testValue");
-						this.matrix.setCell(i, j, sharedString);
+						this.matrix.setCell(i, j, "testValue");
```

Comment:
This feels like it's testing a different thing? DDS-in-DDS vs primitive-in-DDS? Not sure it matters much for functionality, since this is a performance test, but would check with the author if there was a reason to do DDS-in-DDS.

Reply:
To a first approximation, the current Tablero component used by Loop works by inserting a SharedString in every matrix cell.  (It's actually a Scriptor document, but similar).

Is the bug that the benchmark inserted 'sharedString' instead of 'sharedString.handle'?

Reply:
Yes. `.handle` would suffice.
The code also did some funny things with ids. Per the chat there wasn't a need to store SharedString; so I went with simplification. If storing handle is intended - closer mock up; then easy enough to use `.handle` (and I can fix the id collision).
@DLehenbauer, LMK.

Reply:
Changed to keep SharedString.